kie-wb-common-react/import-java-classes/src/index.tsx

```diff
@@ -14,4 +14,15 @@
  * limitations under the License.
  */
 
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { ImportJavaClasses } from "./components";
+
 export * from "./components";
+
+window.renderImportJavaClasses = (selector: string, buttonDisabledStatus: boolean, buttonTooltipMessage?: string) => {
```

Comment:
@yesamer Just a non-blocking comment. This will be slightly different after the React-GWT-integration PR, as we will have a namespace convention to avoid collision of components in the long term. But, it's just something to keep in mind for now :-)